src/AsyncStatusIndicator/AsyncStatusIndicator.tsx

```diff
@@ -27,16 +52,16 @@ export const Props = {
  */
 @skinnable()
 @props(Props)
-export default class AsyncStatusIndicator extends React.PureComponent {
+export default class AsyncStatusIndicator extends React.PureComponent<AsyncStatusIndicatorProps, void> {
```

Comment:
also don't use void, it should be `State extends {}`.
`void` used to break in weird ways with `@types/react@15.0.38`